src/cryptonote_core/blockchain.cpp

```diff
@@ -2606,7 +2609,18 @@ bool Blockchain::check_tx_inputs(transaction& tx, tx_verification_context &tvc,
         return false;
       }
     }
-
+    else 
+    {
+      const size_t mixin = DEFAULT_MIXIN;
```

Comment:
This seems to overwrite the variable `mixin` as calculated above, making the `if` section below a no-op.

Reply:
yes, it overwrites the variable `mixin` (which is set to max) to make sure `mixin` and `mixin_min` always equal `DEFAULT_MIXIN` after v9. I could separate the section to make it cleaner.

Reply:
I don't get it - isn't the idea to reject txes whose mixins are different from the default mixin? Currently it doesn't do so.

Reply:
Currently, users can pick any mixin above the `min_mixin`. After v9, users can only pick the  default mixin.

Reply:
would this make more sense?: https://github.com/wowario/wownero/commit/02ca1bc7b8b12d9587e3609deba6b354ad15ef7d

Reply:
Ah, okay. I see what you mean. Let me try to express it a different way.